extensions/wikia/Wall/WallHistoryController.class.php

```diff
@@ -217,8 +219,18 @@ private function getFormatedHistoryData( $history, $threadId = 0 ) {
 				$history[$key]['msgurl'] = $messagePageUrl;
 
 				$msgUser = $wm->getUser();
+				$msgPage = Title::newFromText( $msgUser->getName(), $ns );
+				if ( empty( $msgPage ) ) {
```

Comment:
Thanks for explanation. 
Having that knowledge I don't think this set of logging parameters will help us to track this issue in future. Instead I suggest checking if HideUser restriction is applied to user and make that clear in the code that we skip revisions that are made by users of that kind. 
What do you think?